build/packer/pmm2.el9.docker.json

```diff
@@ -0,0 +1,184 @@
+{
+  "variables": {
+    "single_disk": "false",
+    "pmm2_server_repo": "testing",
+    "pmm_client_repos": "original testing",
+    "pmm_client_repo_name": "percona-testing-x86_64",
+    "pmm_server_image_name": "perconalab/pmm-server:dev-latest"
+  },
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "ami_name": "PMM2 Server [{{isotime \"2006-01-02 1504\"}}]",
+      "instance_type": "c4.xlarge",
+      "launch_block_device_mappings": [
+        {
+          "delete_on_termination": true,
+          "device_name": "/dev/sda1",
+          "volume_size": 10,
+          "volume_type": "gp2"
+        },
+        {
+          "delete_on_termination": false,
+          "device_name": "/dev/sdb",
+          "volume_size": 100,
+          "volume_type": "gp2"
+        }
+      ],
+      "region": "us-east-1",
+      "security_group_id": "sg-688c2b1c",
+      "source_ami": "ami-0da806cbfc9e67dbb",
+      "ssh_pty": "true",
+      "ena_support": "true",
+      "ssh_username": "ec2-user",
+      "subnet_id": "subnet-ee06e8e1",
+      "run_tags": {
+        "iit-billing-tag": "pmm-ami"
+      },
+      "run_volume_tags": {
+        "iit-billing-tag": "pmm-ami"
+      },
+      "ssh_clear_authorized_keys": "true"
+    },
+    {
+      "type": "azure-arm",
+
+      "subscription_id": "41000701-4126-4674-9219-da03b1f9bb58",
+      "resource_group_name": "percona",
+      "storage_account": "percona",
+      "capture_container_name": "percona",
+      "capture_name_prefix": "pmm2-server-{{ user `pmm_version` }}",
+
+      "os_type": "Linux",
+      "image_publisher": "OpenLogic",
+      "image_offer": "CentOS",
```

Comment:
Is it still "CentOS"?

Reply:
We've a PR to use Oracle Linux 9.2 , at the moment we are unable to test/publish this EL9 based image.

Reply:
What block us from that?